docs/contents/index.md

```diff
@@ -34,6 +34,18 @@ refer to the [Build](users/build.md) instructions to build a cluster from scratc
 See the [Partners](community/partners.md) page for links to third-party methods for
 installing EKS-D.
 
+## Project Tenets
+
+The tenets of the EKS Distro (EKS-D) project are:
+
+1. **The Source**: The goal of the EKS Distro is to be the Kubernetes source for EKS and EKS Anywhere
+2. **Simple**: Make Kubernetes operations, administration, and management simple and boring
+3. **Opinionated Modularity**: Provide opinionated defaults about the best tooling to include with Kubernetes, but give customers the ability to swap them out
+4. **Open**: provide open source tooling backed, validated and maintained by Amazon
+5. **Ubiquitous**: Enable customers to run Kubernetes on the most common public cloud, on-premises bare metal, and on-premises virtualized deployment targets
+6. **Stand Alone**: Provided for use anywhere without AWS dependencies (on-premises air-gapped)
+7. **Better with AWS**: Enable AWS customers to easily adopt additional AWS services
```

Comment:
for the eks-d focused tentets - I think this is not relevant.

Reply:
Lets keep it, we do include AWS-specific components (authenticator) and could include the KMS encryption provider, cloud provider, or others in the future

Reply:
sgtm